### PR TITLE
adds scale queries

### PR DESCRIPTION
**Pull request by hailed748**
Did a lot of work on scaling. At this point it is probably better to have a separate file for each screen size.

### CODE REVIEW

**Comment by Tetheta** [comment]
Starting review

**Comment by Tetheta** [comment]
Nevermind not requested yet. Not sure why it started sending me emails when a PR is created

**Comment by Tetheta** [comment]
Now starting review

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -213,23 +151,28 @@ html {
...
+    width: 40%;
 }
- 
+
 .courses {
     font-size: 18px;
-    line-height: 2;
-    margin-right: 100px;
+    line-height: 1.5;
+    /* margin-right: 100px; */
```
Comment (Tetheta) [inline portfolio/src/main/webapp/style.css:175]: Just remove if not needed

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -654,4 +554,75 @@ span {
...
     width: 190px;
-}
\ No newline at end of file
+}
+
+.margin-fix {
+    margin: auto;
+}
+
+@media screen and (min-width:765px) and (max-width:1900px)
```
Comment (Tetheta) [inline portfolio/src/main/webapp/style.css:563]: Are these like shared styles between the two? Little confused on the empty classnames here. Are you trying to clear the styles above?

Comment (hailed748) [reply portfolio/src/main/webapp/style.css:563]: I forgot to remove this after moving the other file.

**Pull request merged**